=== modified file 'libravatar.py'
--- libravatar.py	2015-06-26 18:25:23 +0000
+++ libravatar.py	2014-09-01 23:28:18 +0000
@@ -165,7 +165,7 @@
     DNS.DiscoverNameServers()
     try:
         dns_request = DNS.Request(name=service_name(domain, https),
-                                  qtype='SRV').req()
+                                  qtype='SRV', protocol='tcp').req()
     except DNS.DNSError as message:
         print("DNS Error: %s" % message)
         return None

